### DISCUSSION TITLE
Unpause Timer

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#25355] [Ideas] [answered]
Recently I've been playing some of the other game modes and overall, they work pretty good, I don't see any glaring issues or anything (But I do mainly play Standard so I'm not exactly the best person to be saying that). But I do have a suggestion.

To start, I'd like to preface by saying I often have to leave my computer in the middle of things. Because of this, I find myself pausing in the middle of maps pretty often. Now, in Standard, there's no problem to doing this, you put your cursor back to where it was before the pause and it's rather easy to get back into the rhythm. But this mode is the only one that has that nice little quality. Every other game mode lacks this. Once you unpause, you are thrown, sometimes violently, back into the gameplay, depending on when you paused. I think implementing a resume timer like Beatsaber has would be beneficial for people like me who have to leave their computer suddenly at any moment.

Like I said at the start, I don't see any actual issues, and unpausing in these modes is fine the way it is, if a little annoying at times.

**Comment by marioman63** [comment]
Most VSRGs I've played have a 3 second countdown upon unpausing. Sometimes a button press will let you skip this countdown. I can't remember if official Taiko games have an unpause count-in but it would make osu! more in line with other similar games.

**Comment by Walavouchey** [comment, marked as answer]
this has been resolved in https://github.com/ppy/osu/pull/27079
